README.md

```diff
@@ -57,7 +57,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: O(n^2) - nested loops -  multiply the n - for each element in words, you loop through each element in endings
```

Comment:
Is it because it's looping through are two different arrays which may have a different number of elements? If that were the case O(n * n) doesn't represent that they are different